drake/examples/Quadrotor/QuadrotorOutput.h

```diff
@@ -79,11 +90,15 @@ bool encode(const double& t, const QuadrotorOutput<double> & x, drake::lcmt_quad
   Eigen::Map<Eigen::Matrix<double, 6, 1>> lcm_twist(msg.twist);
   Eigen::Map<Eigen::Vector3d> lcm_accelerometer(msg.accelerometer);
   Eigen::Map<Eigen::Vector3d> lcm_gyroscope(msg.gyroscope);
+  Eigen::Map<Eigen::Vector3d> lcm_magnetometer(msg.magnetometer);
+  Eigen::Map<Eigen::Matrix<double, x.num_lidar_points, 1>> lcm_lidar(msg.lidar_returns);
```

Comment:
travis complained (legitimately).  this should be QuadrotorOutput::num_lidar_points.